dags/openshift_nightlies/scripts/install/hypershift.sh

```diff
@@ -147,6 +147,22 @@ postinstall(){
     fi
 }
 
+update_fw(){
```

Comment:
Are we doing this for ROSA?

Reply:
I'm not doing this on rosa or self-managed aws clusters, fwiw

Reply:
@smalleni these are needed for uperf testing.

Reply:
@dry923 ive done uperf testing without adding any of these on self-managed OCP on AWS. Are these needed for managed deployments?

Reply:
@smalleni @dry923 Yes and no, these were needed for uperf testing on any manged deployment, but recently we [changed](https://github.com/cloud-bulldozer/benchmark-operator/pull/721/files#diff-c27a1bc402eddb25da91cc4c4072ed6b6760c8a9a742650fe5cff2022fec137f) uperf server ports to listen on `30000`  and there were following update made on [other](https://github.com/cloud-bulldozer/scale-ci-deploy/pull/183/files) places but  here. 
Fortunately there are default rules added by the AWS and GCP to include any port between `30000-32767` so uperf would work without it, but still I am keeping all other rules in place to match `scale-ci-deploy` policies. I can simply rules if needed LMK.